Updating fstab and minio.service path (#936)
minio/docs: source/includes/linux/common-installation.rst

.. start-install-minio-systemd-desc

The ``.deb`` or ``.rpm`` packages install the following `systemd <https://www.freedesktop.org/wiki/Software/systemd/>`__ service file to ``/usr/lib/systemd/system/minio.service``. 
For binary installations, create this file manually on all MinIO hosts.

.. note::
   
   ``systemd`` checks the ``/etc/systemd/...`` path before checking the ``/usr/lib/systemd/...`` path.

>    Make sure the file only exists at ``/usr/lib/systemd/system/minio.service`` path.

Reviewer: It _does_ work with files in multiple locations (`/etc` wins.) Although it's confusing and advising against that is reasonable.

Spitballing:
```suggestion
   To avoid conflicting configurations, ensure this file only exists at ``/usr/lib/systemd/system/minio.service``.
```

The ``.deb`` or ``.rpm`` packages install the following 
`systemd <https://www.freedesktop.org/wiki/Software/systemd/>`__ service file to 

> ``/lib/systemd/system/minio.service``. For binary installations, create this

Reviewer: This explains why my SNSD .deb installs never create this file in /etc. 💡 

But my /etc one is respected, are there rules about which one wins if they both exist?
